bin/pycbc_live

```diff
@@ -1310,18 +1312,27 @@ with ctx:
                     for key in results[ifo]:
                         if len(results[ifo][key]):
                             results[ifo][key] = results[ifo][key][idx]
-                if data_reader[ifo].idq is not None:
-                    logging.info("Checking %s's iDQ information", ifo)
+                if data_reader[ifo].idq:
```

Comment:
Why remove the 'is not None'?

Reply:
I think this was accidental but didn't actually break anything